src/main/java/com/artipie/front/misc/Yaml2Json.java

```diff
@@ -25,12 +25,30 @@ public JsonStructure apply(final String yaml) {
             return Json.createReader(
                 new ByteArrayInputStream(
                     new ObjectMapper().writeValueAsBytes(
-                        new ObjectMapper(new YAMLFactory()).readValue(yaml, Object.class)
+                        new ObjectMapper(new YAMLFactory())
+                            .readValue(Yaml2Json.escapeAsterisk(yaml), Object.class)
                     )
                 )
             ).read();
         } catch (final IOException ex) {
             throw new UncheckedIOException(ex);
         }
     }
+
+    /**
+     * EO yaml {@link com.amihaiemil.eoyaml} does always escapes * while
+     * transforming {@link com.amihaiemil.eoyaml.YamlMapping} into string.
+     * {@link com.fasterxml.jackson.dataformat.yaml.YAMLFactory} does not tolerate it,
+     * so we have to escape.
+     * Asterisk can be met in permissions as item of yaml sequence:
+     * Jane:
+     *   - *
+     * which means that Jane is allowed to perform any actions.
+     * And this "- *" is what we will escape.
+     * @param yaml Yaml string
+     * @return Yaml string with escaped asterisk
+     */
+    private static String escapeAsterisk(final String yaml) {
```

Comment:
I think we can submit a bug to eoyaml about this issue. CC @amihaiemil

Reply:
reported https://github.com/decorators-squad/eo-yaml/issues/493